packages/docs/components/select.md

```diff
@@ -85,7 +85,7 @@ The default Select allows users to choose a single value from a list of options.
         <option value="sirius-a">Sirius A</option>
         <option value="sirius-b">Sirius B</option>
       </select>
-      <label class="ods-label" for="overview-single">Destination star</label>
+      <label class="ods-field--label" for="overview-single">Destination star</label>
```

Comment:
Is this work updating docs for select,radio-button etc an exercise in proving the new format will work?

Reply:
It's both a testbed for the new code structure and the new UI from Design. These will also need to be updated to match the updated format before any release.

Reply:
Related note: unless there's a reason push the work off, I also plan to update the HTML structure in the existing React component once these new styles/structure are finalized.